src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/PointsToAnalysis/PointsToAnalysis.PointsToDataFlowOperationVisitor.cs

```diff
@@ -763,6 +764,23 @@ public override PointsToAbstractValue VisitAnonymousObjectCreation(IAnonymousObj
                 return VisitTypeCreationWithArgumentsAndInitializer(operation, argument, base.VisitAnonymousObjectCreation);
             }
 
+            public override PointsToAbstractValue VisitReDimClause(IReDimClauseOperation operation, object? argument)
+            {
+                var type = operation.Children.FirstOrDefault()?.Type;
+                if (type == null)
+                {
+                    return base.VisitReDimClause(operation, argument);
+                }
+
+                AbstractLocation location = AbstractLocation.CreateAllocationLocation(operation, type, DataFlowAnalysisContext);
+                var pointsToAbstractValue = PointsToAbstractValue.Create(location, mayBeNull: false);
+                CacheAbstractValue(operation, pointsToAbstractValue);
+
+                _ = base.VisitReDimClause(operation, argument);
+
+                return pointsToAbstractValue;
```

Comment:
```suggestion
                return VisitTypeCreationWithArgumentsAndInitializer(operation, argument, base.VisitReDimClause);
```

Reply:
This is causing 2 debug assert to fail, that's why I did this edited copy-paste.